merkle-tree/src/constants/sha256.rs

```diff
@@ -0,0 +1,100 @@
+// This file is generated by xtask. Do not edit it manually.
+
+use super::ZeroBytes;
+pub const ZERO_BYTES: ZeroBytes = [
```

Comment:
Same here we should at least add a comment about the height.

Reply:
Height is dynamic now, so I guess that's solved.